kata-deploy/README.md

```diff
@@ -0,0 +1,115 @@
+# kata-deploy
+
+[kata-deploy](kata-deploy) provides a Dockerfile which contains all of the binaries
+and artifacts required to run Kata Containers, as well as reference daemonsets which can be utilized to install Kata Containers on a running Kubernetes cluster.
+
+Note, installation via daemonsets will only succesfully install `kata-runtime` on
+a node if it is using either containerd or CRI-O CRI-shims.
+
+## Quick start:
+
+### Installing Kata on a running Kubernetes cluster
+
+```
+kubectl apply -f kata-rbac.yaml
```

Comment:
@jodh-intel this can be run without root. root or not depends on how/where the user has setup their `kubeconfig`. The user should run it whichever way they run any other `kubectl` command against their cluster.